### PR TITLE
Serialize numpy array as list

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR fixes a bug in the generic analysis serialization for numpy arrays, serializing them as lists.

### CODE REVIEW

**Comment by stefanv** [comment]
What is this serialization used for? If for JSON, could be okay (depending on dtype), otherwise should use npy format.

**Comment by mcoughlin** [comment]
@stefanv it sends up in a csv file, which renders it with ellipses, so it does not make it with its data to requested analyses.

File: skyportal/handlers/api/analysis.py [hunk 1]
```diff
@@ -580,7 +581,12 @@ def delete(self, analysis_service_id):
 
 class AnalysisHandler(BaseHandler):
     def generic_serialize(self, row, columns):
-        return {c: getattr(row, c) for c in columns}
+        return {
+            c: getattr(row, c).tolist()
+            if type(getattr(row, c)) == np.ndarray
```
Comment (guynir42) [inline skyportal/handlers/api/analysis.py:586]: I would use `isinstance()` to get sub-classes too.

**Review by guynir42 — APPROVED** [review]
LGTM. I think this is a generally good idea. I can imagine other cases where this would blow up but for sure this would catch most of the types of data we are working with.

**Pull request merged**